Makefile

```diff
@@ -1,8 +1,9 @@
-VERSION := $(shell git describe --always --long --dirty --tags)
+VERSION := $(shell git describe --abbrev=0 --tags)
+EXACT_VERSION := $(shell git describe --always --long --dirty --tags)
 GOPATH := $(GOPATH)
 BINARY := rita
 
-LDFLAGS=-ldflags="-X github.com/ocmdev/rita/config.VERSION=${VERSION}"
+LDFLAGS=-ldflags="-X github.com/ocmdev/rita/config.Version=${VERSION} -X github.com/ocmdev/rita/config.ExactVersion=${EXACT_VERSION}"
```

Comment:
VERSION contains the semver
EXACT_VERSION is a full git descriptor for debugging